dev/wiki-visualizer/bin/wiki_visualizer.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:wiki_visualizer/wiki_page.dart';
+import 'package:wiki_visualizer/dot.dart';
```

Comment:
Nit: order alphabetically.

Reply:
weird, why didn't that get caught by the analyzer.... oh, i am probably not analyzing any of this code! let me update the CI stuff and try again.

Reply:
ok added this and the other one in dev/ to `tests.yaml`